src/Evolution/DiscontinuousGalerkin/InitializeElement.hpp

```diff
@@ -274,16 +274,9 @@ struct InitializeElement {
       using Vars = typename variables_tag::type;
```

Comment:
I'm confused by your commit message "set bot tags". What are "bot" tags?

Reply:
I'm guessing a typo for "both"